### PR TITLE
Fix missing conditional execution prefix

### PR DESCRIPTION
**Pull request by EnderShadow**

### CODE REVIEW

File: extensions/variable-width-instruction-standard.md [hunk 1]
```diff
@@ -17,9 +17,10 @@ Instruction prefixes are a sequence of one or more bytes which modify the instru
...
 
-| Prefix      | Description        |
-|:------------|:-------------------|
-| `1010 xxxx` | Unused             |
-| `1100 xxxx` | Expanded registers |
-| `1101 xxxx` | Unused             |
+| Prefix      | Description                                                  |
+|:------------|:-------------------------------------------------------------|
+| `1010 xxxx` | When `xxxx` is neither `1110` nor `1111`, conditional prefix |
+| `1010 1110` | Unused                                                       |
```
Comment (MegaIng) [inline extensions/variable-width-instruction-standard.md:23]: Shouldn't `1010 1111` also be listed here?

File: extensions/stack-and-functions/README.md [hunk 2]
```diff
@@ -32,18 +32,18 @@ This extension adds instructions to allow for convenient use of the stack as wel
...
-| `10 1 0 1111` | `RRR 0 CCCC` | (conditional) absolute register jump                    |
-| `10 1 0 1111` | `RRR 1 CCCC` | (conditional) absolute register function call           |
-| `10 1 0 ????` |              | when `????` is not 1111, reserved for future extensions |
-| `10 1 1 DDDD` | `DDDDDDDD`   | relative unconditional function call                    |
+| First byte    | Second Byte  | Comment                                                       |
+|:--------------|:-------------|:--------------------------------------------------------------|
+| `00 SS 1100`  | `RRR 110 00` | pop from stack                                                |
+| `00 SS 1100`  | `RRR ??? ??` | when `??? ??` is not `110 00`, reserved for future extensions |
+| `00 SS 1101`  | `110 RRR 00` | push register to stack                                        |
+| `00 SS 1101`  | `??? RRR ??` | when `??? ??` is not `110 00, reserved for future extensions  |
```
Comment (MegaIng) [inline extensions/stack-and-functions/README.md]: Here a `` ` `` is missing.

**Review by MegaIng — CHANGES_REQUESTED** [review]

File: extensions/variable-width-instruction-standard.md (hunk 1 above)
Comment (EnderShadow) [reply extensions/variable-width-instruction-standard.md:23]: `1010 1111` isn't a prefix. It's the first byte of the register-based call and jump instructions.

File: extensions/stack-and-functions/README.md (hunk 2 above)
Comment (EnderShadow) [reply extensions/stack-and-functions/README.md]: Good catch. Fixed

File: extensions/variable-width-instruction-standard.md (hunk 1 above)
Comment (MegaIng) [reply extensions/variable-width-instruction-standard.md:23]: Aha, right

**Review by MegaIng — APPROVED** [review]

**Pull request merged**